### DISCUSSION TITLE
Features wanted

### DISCUSSION THREAD

**Discussion by andreas-nn** [HASwitchPlate/openHASP#456] [Ideas]
openHASP is a excellent project. (But all, what is good, can become better.)

My ideas/wishes for new features:

- new object like gauge, but linear: Like a thermometer with scale.
- aliases : I would like to use "temperature" instead of "p2b10".
- real datatype
- calculating in values (addition, multiplication, value to string)

- rtc-variables (%date%, %time%) to make a clock
- trigger an action/run-script by changing of any value (like Issue #454)
- option to disable the touch. to make read-only-displays
...

All above is can also be realized by the homeautomation, but it would be better, if I could make it direct in openHASP.

With the best wishes to your work! Thanks.
Andreas

**Comment by adorobis** [comment]
> * option to disable the touch. to make read-only-displays

I believe this is already in place - if you don't program any responses to events (in HA integration config) it will be a read only display.
